### ISSUE TITLE
purchasePackage(package) is crashing on android

### ISSUE THREAD

**Issue by EdesonABizerril** [RevenueCat/purchases-flutter#244]
- [x] I have updated Purchases SDK to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/RevenueCat/purchases-flutter/blob/main/CONTRIBUTING.md)
- [x] I have read the [Help Center](https://support.revenuecat.com/hc/en-us)
- [x] I have read [docs.revenuecat.com](https://docs.revenuecat.com/)
- [x] I have searched for [existing Github issues](https://github.com/RevenueCat/purchases-flutter/issues)

**Describe the bug**
When I try to make a purchase, the sales modal opens, but it remains locked until the back button is activated..

1. Environment
In Pubspeck file: `purchases_flutter: 3.4.5`

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.0.6, on Mac OS X 10.15.7 19H2 darwin-x64, locale pt-BR)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[!] Android Studio (version 2020.3)
✗ Unable to find bundled Java version.
[✓] VS Code (version 1.60.1)
[✓] Connected device (4 available)
```

! Doctor found issues in 1 category.

2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
There is no reaction in the log during the crash. I didn't run it using --verbose. If you need to, let me know.

3. Steps to reproduce, with a description of expected vs. actual behavior
Try to make a purchase, as shown in the code below.

4. **Other information** (e.g. stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc.)

```
Offerings _offerings = await Purchases.getOfferings();
      Package package = _offerings.all[toolId.id].availablePackages
          .firstWhere((package) => package.product.identifier == identifierProduct);
      PurchaserInfo purchaserInfo = await Purchases.purchasePackage(package);
      if (purchaserInfo.entitlements.all[identifierProduct].isActive) {
        print("purchaserInfo: ${purchaserInfo.toString()}");
      }
```
**Additional context**
   I tested it on android 6 and 11. Here's my data:

**Comment by vegaro** [comment]
Hmm, this is weird. The sales modal is controlled by Google not us. Can you [enable debug logs](https://docs.revenuecat.com/docs/debugging) and share the logs you get in Logcat in Android Studio?

Thanks!

**Comment by EdesonABizerril** [comment]
@vegaro I just discovered the problem, completely unusual. I use a screen mirroring software on Mac called Wormhole, I tried to use it without using it and it worked correctly.
My suspicion came from the moment I used a version that I was sure worked, v3.2.2, and the same problem occurred. So, not using Wormhole was the first idea I had and it worked perfectly, even in version 3.4.5.
Thank you very much for your attention. Now I will contact Wormhole support =)
